### PR TITLE
SCons: Pass `ninja_file` variable in tool instead of default name

### PR DESCRIPTION
**Pull request by ghost**
Noticed that when I run with `ninja_file` it wasn't created even though in terminal I saw that the file will be generated

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -36,7 +36,7 @@ compile_commands.json
 platform/windows/godot_res.res
 
 # Ninja build files
-build.ninja
+*.ninja
 .ninja
```
Comment (akien-mga) [inline .gitignore:40]: `*` also covers 0 chars, so the second one is redundant:
```suggestion
*.ninja
```

**Review by akien-mga — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (dustdfg) [reply .gitignore:40]: Second one is directory so I decided to leave just in case

Comment (akien-mga) [reply .gitignore:40]: It seems to be properly ignored with my suggested change, but yeah we could keep it explicit, then I'd go one step further and make it `.ninja/` so it's clear it's a directory.

**Comment by Repiteo** [comment]
Thanks!

**Pull request merged**